homework/index.js

```diff
@@ -30,13 +30,108 @@
     return elem;
   }
 
+  function update(val, dBase) {
```

Comment:
What is the meaning of these names: `val` and `dBase`? I need to deduce from looking into your code. I see in line 38 that you use these params to get the repository info to update. And this line 38 is the only place where you use these parameters. It would be better to have just one parameter, `repository`, and call this function with:

```js
update(repos[selector.value]);
```

The `update` function has no need to have access (and therefore should not have access) to the array with **all** repositories. The only one it needs to know about is the one it needs to update. Supplying data on a 'need to know' basis will help to make your code more robust.

Reply:
val : represents value of select element. 
dBase : represent database (all repos).
I understood it is better if i pass just one repo to update function. 
First, I wrote all codes in initialize then i realized I should create a function to use in case of changing select element. Then I create this function. But it makes me blind to realize these things. 
Thanks for advices.